packages/server/graphql/public/types/ToggleFeatureFlagSuccess.ts

```diff
@@ -0,0 +1,18 @@
+import {ToggleFeatureFlagSuccessResolvers} from '../resolverTypes'
+
+export type ToggleFeatureFlagSuccessSource = {
+  featureFlagId: string
+  enabled: boolean
+}
+
+const ToggleFeatureFlagSuccess: ToggleFeatureFlagSuccessResolvers = {
+  featureFlag: async ({featureFlagId, enabled}, _, {dataLoader}) => {
+    const flag = await dataLoader.get('featureFlags').loadNonNull(featureFlagId)
+    return {
+      ...flag,
+      enabled
```

Comment:
+1 is this the only place where `enabled` is put onto the feature flag object? if so, it might make more sense to put `enabled` on its own field